src/classMembersNameRule.ts

```diff
@@ -300,7 +293,7 @@ class ClassMembersWalker extends Lint.ProgramAwareRuleWalker {
                     name.getText()
                 ))
         ) {
-            this.checkNameNode(name, format || Format.None, leadingUnderscore);
+            this.checkNameNode(name, format || Format.None, prefix);

```

Comment:
Maybe move `|| Format.None` as a fallback in `format` declaration and change format type to `Format`?

Reply:
Fixed.